samza-sql-shell/src/main/java/org/apache/samza/sql/client/cli/CliEnvironment.java

```diff
@@ -173,11 +187,21 @@ public SqlExecutor getExecutor() {
     return executor;
   }
 
+  public List<CommandHandler> getCommandHandlers() { return commandHandlers; }
+
   private void createShellExecutor(String executorClassName) throws ExecutorException {
+    executor = (SqlExecutor) createInstance(executorClassName);
+  }
+
+  private void createCommandHandler(String handlerClassName) throws ExecutorException {
```

Comment:
Throwing a `ExecutorException` here may not be appropriate.

Reply:
changed it to throw CommandHandlerException